doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+# Controls and Shortcuts #
+
+The following is a list of input device controls and shortcuts for manipulating Gaffer's _Node Graph_, _Viewer_, and _Script Editor_ tabs.
+
+
+## _Node Graph_ Tab ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover inside the _Node Graph_ tab.
+
+### Node Creation ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Show node menu                        Right-click
+"                                     :kbd:`Tab`
+Insert `Dot` at connection            Right-click on connection > `Insert Dot`
+===================================== =============================================
+```
+
+### Node Selection ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Select all                            :kbd:`Ctrl`-:kbd:`A`
```

Comment:
should this (and all the others) be Ctrl+ rather than Ctrl-?

Reply:
There's differing opinions on this, but a dash is the most common. I think dashes add less visual clutter than pluses, making for easier skimming.

Reply:
I think it's more important that the format used in the docs matches the format used in the UI. On Linux this appears to be + (it's not something I've chosen specifically, it's Qt's choice). Actually, I've just been through a bunch of apps on my Linux machine and they're all using +. On Mac, there appears to be no + or -, they just use a symbol for Shift and use the funky Apple symbol instead of Ctrl.

Reply:
I think I've been voted off the island!

I'm personally fond of the [Apple online help's style choices](https://support.apple.com/en-us/HT201236), but I'll defer to what we're already using and what users are expecting.